src/backend/distributed/master/master_truncate.c

```diff
@@ -0,0 +1,86 @@
+/*-------------------------------------------------------------------------
+ *
+ * master_truncate.c
+ *
+ * Routine for truncating local data after a table has been distributed.
+ *
+ * Copyright (c) 2014-2017, Citus Data, Inc.
+ *
+ *-------------------------------------------------------------------------
+ */
+
+#include "postgres.h"
+#include "c.h"
```

Comment:
c.h shouldn't be included from backend programs (yes, we do that occasionally, but we shouldn't).